src/NBitcoin/RPC/IRPCClient.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NBitcoin.RPC
+{
+    /// <summary>
+    /// Interface for an RPC client.
+    /// </summary>
+    /// <remarks>Extend where necessary.</remarks>
+    public interface IRPCClient
+    {
+        /// <summary>Send a command over RPC.</summary>
+        /// <param name="commandName">The command to execute.</param>
+        /// <param name="parameters">An array of command parameters.</param>
+        /// <remarks>See https://en.bitcoin.it/wiki/Original_Bitcoin_client/API_calls_list </remarks>
+        /// <returns>The RPC response.</returns>
+        RPCResponse SendCommand(RPCOperations commandName, params object[] parameters);
+
+        /// <summary>Send a command over RPC.</summary>
+        /// <param name="rpcRequest">The rpc request to execute.</param>
```

Comment:
param name does not represent the input variable name. Change here to 'request' or change 'request' to rpcRequest at 26

same on line 43